### ISSUE TITLE
Anchors should be anchors, links should be links, and includes should be includes.

### ISSUE THREAD

**Issue by leoj3n** [OscarGodson/HTML6#5]
_Everyday terminology_ refers to the semantic `<a>` element as a "link" _OR_ an "anchor", giving the tag a double meaning.

Specs refer to a "link" as what _everyday terminology_ refers to as an "include".

This is a serious barrier of entry for anyone trying to learn HTML. "I've been told links are what take you to new documents my whole life, but why are they something completely different in HTML land?", students ask. "Because the HTML gods made it so", I tell them.

Everyday peoples understand the (HUGE) difference between a website "link" and an "anchor". It's an easy concept, conceptually. However, once they get into HTML, real-world understanding doesn't map to the reality ( _cough_ poorly invented conventions _cough_ ) of HTML-fairytale-land.

I've always had this question, but figured there was some mystical force at work that justified such idiotic terminology. After exploring the deepest caverns of HTML I can finally put that myth to rest, and take a stand against this retardation!

A ludicrous proposal: a "link" actually be what normal people understand a "link" to be. This means `<html:a>` or possibly `<html:anchor>` will only be used for ANCHORS and `<html:link>` will only be used for LINKS in the true sense of the term. Now that `<html:link>` is taken, we must endow a new term for the _inclusion_ of source files; I propose `<html:include>`.

Please poke holes in this philosophy if you can, because from where I sit it seems sound; but perhaps I missed one of those secret underground passageways that leads you to the bonus level where there's flying pigs and this all makes sense.

PS: "HTML6" seems like a splendid idea, regardless of if my points are valid.

**Comment by OscarGodson** [comment]
Yeah, totally, `<a>` isn't very semantic at all when you're not talking about an anchor and I've always been curious about that too, "why is an anchor also a link?". I'm not sure I follow the `include` idea tho. Include implies including the source of something on the page. Like if it were:

```
<p>
<html:include src="/helloworld.html"></html:include>
</p>
```

I'd assume it was going to put the content of /helloworld.html on the page between the `<p>` tags since the keyword include is used in programming languages that way.

**Comment by icalvo** [comment]
`<html:import>` or `<html:using>`?

**Comment by cshaa** [comment]
It could be this way:
`<html:link>` for hyperlink to another page
`<html:anchor>` for anchor (but maybe it's too long, `anch` or `a` could be enough)
`<html:using>` for scripts and style sheets
`<html:alter>` for alternatives like RSS or Atom feeds

**Comment by cshaa** [comment]
And what about removing `html:a` completely - leaving just `href` or `html:href` attributes would be great. Even W3C was trying to do this in their [XHTML2](http://en.wikipedia.org/wiki/XHTML#XHTML_2.0). Less tags, less writing.
